src/app/zap-templates/zcl/zcl.json

```diff
@@ -298,7 +299,8 @@
             "LocalTime"
         ],
         "Temperature Control": ["SupportedTemperatureLevels"],
-        "Operational State": ["OperationalState", "OperationalError"]
+        "Operational State": ["OperationalState", "OperationalError"],
+        "Dishwasher Operational State": ["OperationalState", "OperationalError"]
```

Comment:
Is there are reason these two lines were added here? 

I don't see similar changes in the Mode Select XML PR:
https://github.com/project-chip/connectedhomeip/pull/26508